README.md

```diff
@@ -56,6 +56,13 @@
 
 ## Dependencies
 
+### SDK
+
+The Android application consumes the internal [Bitwarden SDK](https://github.com/bitwarden/sdk), by
+default it consumes the published artifacts however by setting `localSdk` in `user.properties` you
+can configure it to load the SDK from your local maven artifactory. This is particularly useful when
+developing new SDK capabilities.
```

Comment:
Step 2 above in `Setup` talks about the different properties that should be placed in the `user.properties` file. It might make sense to move these docs there and call out `localSdk` as an optional property.

Also, can we link to https://sdk.contributing-docs.pages.dev/getting-started/sdk/#linking-sdk-to-clients in these docs to make clear how the local maven artifactory gets populated?